### PR TITLE
[repo] Clarify resource detectors README

### PR DESCRIPTION
**Pull request by joegoldman2**
Addresses https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1943#discussion_r1674273335.
Fixes: #1509

### CODE REVIEW

File: src/OpenTelemetry.Resources.AWS/README.md [hunk 1]
```diff
@@ -20,18 +20,21 @@ dotnet add package OpenTelemetry.Resources.AWS
...
-var tracerProvider = Sdk.CreateTracerProviderBuilder()
-                        // other configurations
-                        .SetResourceBuilder(ResourceBuilder
-                            .CreateEmpty()
-                            .AddAWSEC2Detector())
-                        .Build();
+using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+    .ConfigureResource(resource => resource.AddAWSEC2Detector())
+    // other configurations
+    .Build();
```
Comment (Kielek) [inline src/OpenTelemetry.Resources.AWS/README.md:32]: Could you please also add example for logs (all readme files)?

Comment (Kielek) [reply src/OpenTelemetry.Resources.AWS/README.md:32]: Ref: #1509 and https://github.com/open-telemetry/opentelemetry-dotnet/pull/5188

Comment (joegoldman2) [reply src/OpenTelemetry.Resources.AWS/README.md:32]: Done. Do you think this PR fixes #1509?

**Review by Kielek — APPROVED** [review]

File: src/OpenTelemetry.Resources.AWS/README.md (hunk 1 above)
Comment (Kielek) [reply src/OpenTelemetry.Resources.AWS/README.md:32]: Yes! Edited first comment to automatically close that issue.

**Review by cijothomas — APPROVED** [review]
Thanks for improving the docs!

**Pull request merged**